src/contractTest/java/uk/gov/hmcts/probate/services/consumer/ServiceAuthenConsummerDetailTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.probate.services.consumer;
+
+import au.com.dius.pact.consumer.dsl.PactDslRootValue;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@PropertySource(value = "classpath:application.yml")
+@EnableAutoConfiguration
+@ExtendWith(PactConsumerTestExt.class)
+@ExtendWith(SpringExtension.class)
+@PactTestFor(providerName = "s2s_auth", port = "4502")
+@SpringBootTest(classes = ServiceAuthorisationApi.class)
+public class ServiceAuthenConsummerDetailTest {
+
```

Comment:
maybe name ServiceAuthConsumerDetailTest - as we usually shorten authentication to auth and consummer is mispelling of consumer